### PR TITLE
Guideline for Android developers

### PR DESCRIPTION
**Pull request by thamha-emotiv**
I didn't finish the guideline, but I want to receive early comments from our team, so feel free to leave any comments or suggestions. Thanks!

### CODE REVIEW

File: Android/CortexV2Example/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+
+## Prequisitions
```
Comment (bchaperon) [inline Android/CortexV2Example/README.md]: The word you are looking for is "Prerequisite", not "Prequisitions" :)

File: Android/CortexV2Example/README.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+- You can click on the link provided in the email, follow the guide to install EMOTIV App on your Android device.
+	- You cannot install if you have any version of EMOTIV App installed on your device, so make sure you uninstall it (for all users) before installing a new version.
+	- If you send the link to others, they may not access to our unreleased version as long as they are not in our list.
+
+##### Released version
+
+Not available yet.
+
+## Try EMOTIV App
+- Open EMOTIV App, make sure you accept all permission request pop ups.
```
Comment (tuongtiv) [inline Android/CortexV2Example/README.md]: Should we add one more line: "Make sure Bluetooth & Location are turned-on"?

File: Android/CortexV2Example/README.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+- Log in using your Emotiv account. You are on the page of "Available Devices".
+- Turn on your Emotiv headset, wait for the headset to show up.
+	- *Note: If you don't see the headset after 30s, you can try to turn it off then on again.*
+- After the headset is shown up on the list, click to connect.
```
Comment (tuongtiv) [inline Android/CortexV2Example/README.md]: do we need to note users that our system just supports to connect to only one headset at a moment?

**Review by nguoithichkhampha — APPROVED** [review]

File: Android/CortexV2Example/README.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+- You can navigate to other pages using the top left menu icon.
+- If you see anything inconvenience, you can find `Send Feedback` in the menu.
+
+## Try Cortex V2 Example
```
Comment (nguoithichkhampha) [inline Android/CortexV2Example/README.md]: we should warning developer. before run example, they need to set their client_id/secret in file 
`cortexclient/src/main/java/com/emotiv/cortexv2example/utils/Constant.java`

File: Android/CortexV2Example/README.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
+
+## Prerequisites
```
Comment (nguoithichkhampha) [inline Android/CortexV2Example/README.md]: you should require to have the client id/secret of app as well
`- The client id/secret of your cortex app (if you don't have, you can register [here](https://www.emotiv.com/my-account/cortex-apps/))`

**Pull request merged**